dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java

```diff
@@ -184,8 +188,17 @@ private String buildCustomConfigContent() {
     }
 
     private String buildConfigFilePath() {
-        return String.format("%s/seatunnel_%s.conf", taskExecutionContext.getExecutePath(),
-                taskExecutionContext.getTaskAppId());
+        return String.format("%s/seatunnel_%s.%s", taskExecutionContext.getExecutePath(),
+                taskExecutionContext.getTaskAppId(), formatDetector());
+    }
+
+    private String formatDetector() {
+        try {
+            new Gson().fromJson(seatunnelParameters.getRawScript(), JsonElement.class);
+            return "json";
+        } catch (JsonSyntaxException e) {
```

Comment:
If it is not `json`, are there any other format besides `conf`?

Reply:
No, currently only these two formats are available

Reply:
https://seatunnel.apache.org/docs/2.3.4/concept/config